pandas/tests/series/methods/test_values.py

```diff
@@ -27,3 +28,12 @@ def test_values(self, datetime_series):
         tm.assert_almost_equal(
             datetime_series.values, datetime_series, check_dtype=False
         )
+
+    def test_values_multiplying_large_series_by_NA(self):
+        # GH#33701
+        Int64NA = Series([np.nan], dtype=Int64Dtype()).iloc[0]
```

Comment:
You can just replace `Int64NA` with `pd.NA`

Also can you move to `pandas/tests/array/integer/test_arithmetic.py`?